home/tests.py

```diff
@@ -1,3 +1,40 @@
-# from django.test import TestCase
+from django.contrib.auth.models import User
+from home.models import Profile, Question, Answer
+import pytest
 
-# Create your tests here.
+
+@pytest.mark.django_db
+def test_Add_Profile():
```

Comment:
If you have a profile creation class method, what is the poing of rewriting profile creation code is a part of a test? What is this testing exactly?